inferno-framework/us-core-test-kit
# FI-1903 Remove device-udi-pi test

# Summary
This PR fix GitHub Issue https://github.com/onc-healthit/onc-certification-g10-test-kit/issues/404

This PR changes:
* Remove UDI-PI test from US Core Test Kit v4.0.0 and v5.0.1
* Remove corresponding template and unit test

# Testing Guidance
* Start US Core v4.0.0 Test Kit
* Verify that test 7.08 does not exist
* Start US Core v5.0.1 Test Kit
* Verify that test 8.08 does not exist
